donate/templates/pages/core/contributor_support_page_master.html

```diff
@@ -24,20 +30,22 @@ <h1 class="heading heading--primary">
                     <p class="rich-text">{% trans "We will get back to you soon." %}</p>
                 </div>
             {% else %}
-                <form class="" action="{{ salesforce_form_url }}" method="POST">
+                <form class="" action="https://webto.salesforce.com/servlet/servlet.WebToCase?encoding=UTF-8" method="POST">
                     <p class="rich-text">{% blocktrans trimmed %}
                     If you need help with a contribution to MZLA/Thunderbird, please fill out this form and a Thunderbird donor care representative will get back to you as soon as possible.
                     {% endblocktrans %}</p>
 
-                    <p class="rich-text">{% blocktrans with support_url="https://support.mozilla.org/products/thunderbird/" trimmed %}
-                    Unfortunately, donor care representatives are unable to offer support or help with Thunderbird technical issues. For technical support, please visit our <a href="{{ support_url }}">support page.</a>
-                    {% endblocktrans %}</p>
+                    {% if 'thunderbird' in current_site.hostname %}
+                        <p class="rich-text">{% blocktrans with support_url="https://support.mozilla.org/products/thunderbird/" trimmed %}
+                        Unfortunately, donor care representatives are unable to offer support or help with Thunderbird technical issues. For technical support, please visit our <a href="{{ support_url }}">support page.</a>
+                        {% endblocktrans %}</p>
+                    {% endif %}
```

Comment:
If the site is a thunderbird site show the help text. Otherwise don't show the Thunderbird-specific help text for the donate site.

Reply:
Unless I’m missing something, I think we use an empty block here and move the TB copy to an override block in https://github.com/mozilla/donate-wagtail/blob/master/donate/thunderbird/templates/pages/core/contributor_support_page.html

Reply:
Hey @TheoChevalier Thanks for looking this over! 

I tried to do this this morning, and got nowhere. It actually looks like that file you linked to isn't being used at all.  At least not on localhost when I set the site up to be a thunderbird site. This might be something we should look into in another ticket. For now I'm going to opt for the `if` statement